src/main/java/com/example/QuestionPortalBackend/util/QuestionMapper.java

```diff
@@ -1,5 +1,67 @@
 package com.example.QuestionPortalBackend.util;
 
-public class QuestionMapper {
-    //TODO Create mapper
+import com.example.QuestionPortalBackend.dto.QuestionDTO;
+import com.example.QuestionPortalBackend.models.Question;
+import com.example.QuestionPortalBackend.models.QuestionOption;
+import com.example.QuestionPortalBackend.models.User;
+import com.example.QuestionPortalBackend.services.UsersService;
+import org.springframework.stereotype.Component;
+
+import javax.swing.text.html.Option;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+public class QuestionMapper implements MapperInterface<Question, QuestionDTO>{
+    private final UsersService usersService;
```

Comment:
it's not a best practice to use services inside mappers. Mappers should be used by services, not vice versa